src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowCreate.cs

```diff
@@ -130,9 +130,13 @@ public override DbCommand GetCommand(DbConnection connection)
         public override EditRow GetEditRow(DbCellValue[] cachedRow)
         {
             // Iterate over the new cells. If they are null, generate a blank value
-            DbCellValue[] editCells = newCells.Select(cell => cell == null
-                    ? new DbCellValue {DisplayValue = string.Empty, IsNull = false, RawObject = null}
-                    : cell.AsDbCellValue)
+            EditCell[] editCells = newCells.Select(cell =>
+                {
+                    DbCellValue dbCell = cell == null
+                        ? new DbCellValue {DisplayValue = string.Empty, IsNull = false, RawObject = null}
```

Comment:
Yes, if we return null, it indicates that the value is null and should apply the null styling.